alfresco-solr-query-analytics/src/main/amp/config/alfresco/module/alfresco-solr-query-analytics/log4j.properties

```diff
@@ -14,5 +14,5 @@ log4j.appender.solrQueryLog.CompressBackups=true
 log4j.logger.eu.xenit.alfresco.instrumentation.solr.DebugSolrQueryHTTPClientAdvisor =debug, solrQueryLog
 log4j.additivity.eu.xenit.alfresco.instrumentation.solr.DebugSolrQueryHTTPClientAdvisor=false
 
-#Enables logger on our overwritten custon class, effectively shows if our function is being called.
-log4j.logger.org.alfresco.repo.search.impl.lucene.SolrJSONResultSet=debug
\ No newline at end of file
+#Enables logger on our overridden class, effectively shows if our new function is being called.
+log4j.logger.org.alfresco.repo.search.impl.lucene.SolrJSONResultSet=warn
```

Comment:
non-blocking remark: In my opinion, this can be removed.

Reply:
Hmmmm, agreed, I only had this in to aid troubleshooting, I'll remove it for clarity.